lib/shared/bucketing-assembly-script/assembly/index.ts

```diff
@@ -31,3 +32,10 @@ export function testBucketedUserConfigClass(userConfigStr: string): string {
 export function testMurmurhashV3(key: string, seed: i32): i32 {
     return murmurhashV3(key, seed)
 }
+
+class TestData {
+    key: string
+}
+export function testSortObjectsByString(arr: SortingArray<TestData>): TestData[]{
```

Comment:
does calling this like this from JS actually work without JSON parsing?

Reply:
interested in what it looks like in: `bucketing-lib.debug.d.ts` / `bucketing-lib.debug.js`

Reply:
It seems to output a function that receives an array, but either way the tests are passing so that must mean it works?